app/services/requisicion_service.py

```
@@ -0,0 +1,536 @@
+                    agregar_producto_al_catalogo(nombre_producto_estandarizado)
+
+            db.session.commit()
+
+            # Notificaciones y PDF post-commit
+            try:
+                mensaje = generar_mensaje_correo('Solicitante', nueva_requisicion, nueva_requisicion.estado, "")
+                enviar_correo([nueva_requisicion.correo_solicitante], 'Requisición creada', mensaje)
+
+                if nueva_requisicion.estado == ESTADO_INICIAL_REQUISICION:
+                    mensaje_almacen = generar_mensaje_correo('Almacén', nueva_requisicion, nueva_requisicion.estado, "")
+                    enviar_correos_por_rol('Almacen', 'Nueva requisición pendiente', mensaje_almacen)
```

Pylint (3.3.7)

[error] No value for argument 'UsuarioModel' in function call

(E1120)